File: ruby/spec/pos_spec.rb
require 'rspec'
require './pos'

describe Pos do
    describe '.barcode' do
        it 'returns a price when a matching barcode is found' do
            pos = Pos.new()
            pos.barcode('1234567890')

            expect(pos.message).to eq '$1.00'
        end
    end
end

File: node/test/pos.spec.js
const poc = require("../pos");
const expect = require("chai").expect;

describe("pos", () => {
  it("returns a price if a matching barcode is found", () => {
    pos.onBarcode("1234567890");
    expect(pos.lastMessage).to.equal("$1.00")
  });
});
